src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -82,4 +94,62 @@ public String getSignerName() {
     public String getSignerRole() {
         return signerRole;
     }
+
+    private void setAmountToPayByDefendant(
+        final InterestCalculationService interestCalculationService,
+        final Claim claim
+    ) {
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        if (countyCourtJudgment.getPaymentOption() == PaymentOption.INSTALMENTS) {
+            countyCourtJudgment.getRepaymentPlan().ifPresent(
+                repaymentPlan -> this.amountToPayByDefendant = formatMoney(repaymentPlan.getRemainingAmount())
+            );
+        } else if (countyCourtJudgment.getPaymentOption() == PaymentOption.FULL_BY_SPECIFIED_DATE
+            || countyCourtJudgment.getPaymentOption() == PaymentOption.IMMEDIATELY) {
+            this.amountToPayByDefendant = calculateRemainingAmount(
+                interestCalculationService,
+                claim,
+                claim.getCountyCourtJudgmentRequestedAt().toLocalDate()
+            );
+        }
+    }
+
+    private String calculateRemainingAmount(
+        final InterestCalculationService interestCalculationService,
+        final Claim claim,
+        final LocalDate toDate
+    ) {
+        BigDecimal claimAmount = ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount();
+        BigDecimal paidAmount = claim.getCountyCourtJudgment().getPaidAmount() != null
```

Comment:
isn't this defaults to zero?

Reply:
no it was null and was throwing a null pointer exception when I was testing it

Reply:
I think we need to default it to zero